stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/dto/DownstreamMessageDto.kt

```diff
@@ -0,0 +1,20 @@
+package io.getstream.chat.android.client.api2.model.dto
+
+import com.squareup.moshi.JsonClass
+import java.util.Date
+
+@JsonClass(generateAdapter = true)
+internal data class DownstreamMessageDto(
+    val id: String,
+    val cid: String,
+    val text: String,
+    val html: String,
+    val parent_id: String?,
```

Comment:
We are doing that to avoid using `@Filed` right? IMO it would be better to keep it in camel case as we do everywhere else (even with views' ids). Does Moshi provide something similar to `GsonBuilder:: setFieldNamingPolicy `?

Reply:
Moshi does _not_ have a field naming policy, and we should be really happy about that! 😄 

Compared to Gson, from [their README](https://github.com/square/moshi#borrows-from-gson):

>Moshi is less configurable. There’s no field naming strategy, versioning, instance creators, or long serialization policy. Instead of naming a field `visibleCards` and using a policy class to convert that to `visible_cards`, Moshi wants you to just name the field `visible_cards` as it appears in the JSON.

Some reasons why this is great:

- Code is more straightforward, less magic involved
- No ugly `@SerializedName` annotations making the classes unreadable
- You can search for the JSON field's name in the project and find the corresponding property
- It makes you not want to use these fields in your code because they're ugly in Kotlin, encouraging you to not use your network models all over the place, but to map them to domain models instead

Reply:
Fair enough! Thanks for great explanation